src/app/component/core/search-popup/search-popup.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { SearchClickService } from '../../../service/search-click/search-click.service';
+
+@Component({
+  selector: 'app-search-popup',
+  templateUrl: './search-popup.component.html',
+  styleUrls: ['./search-popup.component.scss']
+})
+export class SearchPopupComponent implements OnInit {
+  private _isSearchClicked = false;
+  private isSearchFound: boolean = true;
+
+  set isSearchClicked(value: boolean) {
+    this.onSearchClick();
```

Comment:
Please, delete it. You should have the possibility to scroll. And using 'document.body..... ' instead of @ViewChild is a bad practice

Reply:
Delete line 14 and appropriate method

Reply:
fixed